template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/resources/monitoring-workflow-resource.yml.tmpl

```diff
@@ -1 +1,59 @@
-# TODO: Add data monitoring support for mlops
+# Note: Lakehouse monitring only works for unity catalog enabled workspaces. Remove this workflow if your application
```

Comment:
I think we should combine these two resource files so they only have to look at one for TODOs